### PR TITLE
Request received

### PR DESCRIPTION
**Pull request by heshambadawy**

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 export const environment = {
   production: false,
-  apiUrl :  "https://www.aychihuahua.org"
+  // apiUrl :  "https://www.aychihuahua.org"
+  apiUrl :  "http://localhost:3001"
```
Comment (Aychihuahuausa) [inline src/environments/environment.ts:8]: don't push local enviroment

File: src/global.scss [hunk 2]
```diff
@@ -24,3 +24,70 @@
...
+.notification-modal .modal-wrapper{
+    --border-radius: 1rem;
+    --width: 97%;
+    border-radius: 1rem !important;
+    position: absolute;
+    height: 48vh;
+    top: 38vh;
+}
+
+// @media screen
```
Comment (Aychihuahuausa) [inline src/global.scss:38]: remove dead code

File: src/app/main-client/main-client.component.ts [hunk 3]
```diff
@@ -62,11 +124,11 @@ export class MainClientComponent implements OnInit {
         }, 200);
       });
     })
-    this.userService.getCurrentWalker().subscribe(x => {
-      if (x) {
-        this.userService.currentWalker = x;
-      }
-    });
+    // this.userService.getCurrentWalker().subscribe(x => {
```
Comment (Aychihuahuausa) [inline src/app/main-client/main-client.component.ts:127]: why u commented get walker

File: src/app/main-client/main-client.component.ts [hunk 4]
```diff
@@ -36,13 +99,13 @@ export class MainClientComponent implements OnInit {
...
       .subscribe((data: any) => {
-        if (data.coords && userService.currentWalker) {
-          mainClientService.updateWalkerPosition({ lat: data.coords.latitude, lng: data.coords.longitude },
-            userService.currentWalker.walkerId).subscribe()
+        if (data.coords && this.userService.currentWalker) {
+          this.mainClientService.updateWalkerPosition({ lat: data.coords.latitude, lng: data.coords.longitude },
+            this.userService.currentWalker.walkerId).subscribe()
         }
       });
     this.mapService.mapCenterChanged.subscribe(x => {
```
Comment (Aychihuahuausa) [inline src/app/main-client/main-client.component.ts:111]: plz remove subscribe if it do noting

File: src/app/main-client/main-client.component.ts [hunk 5]
```diff
@@ -52,7 +115,6 @@ export class MainClientComponent implements OnInit {
     })
   }
 
-
   ngOnInit(): void {
     this.mapService.mapLoaded.subscribe(x => {
```
Comment (Aychihuahuausa) [inline src/app/main-client/main-client.component.ts:119]: plz remove subscribe if it do noting

File: src/global.scss (hunk 2 above)
Comment (heshambadawy) [reply src/global.scss:38]: please let them i will utilize them later

File: src/app/main-client/main-client.component.ts (hunk 4 above)
Comment (heshambadawy) [reply src/app/main-client/main-client.component.ts:111]: I don't make that part i have no clue about , i think it's yours

File: src/app/main-client/main-client.component.ts (hunk 5 above)
Comment (heshambadawy) [reply src/app/main-client/main-client.component.ts:119]: same here i have no clue

**Pull request merged**